ui_framework/doc_site/src/views/popover/popover.js

```diff
@@ -0,0 +1,48 @@
+import React, {
+  Component,
+} from 'react';
+
+import {
+  KuiPopover,
+  KuiButton,
+} from '../../../../components';
+
+export default class extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isPopoverOpen: false,
+    };
+  }
+
+  onButtonClick() {
+    this.setState({
+      isPopoverOpen: !this.state.isPopoverOpen,
+    });
+  }
+
+  closePopover() {
+    this.setState({
+      isPopoverOpen: false,
+    });
+  }
+
+  render() {
+    const button = (
+      <KuiButton onClick={this.onButtonClick.bind(this)}>
```

Comment:
Could we add `buttonType="basic"` to the Button instances in these examples, just so they don't look funny?